.github/workflows/release.yml

```diff
@@ -22,6 +22,6 @@ jobs:
       - run: npm install
       - run: npm run build
       - run: npm config set '//registry.npmjs.org/:_authToken' "${{ secrets.NPM_TOKEN }}"
-      - run: npm publish --tag ${{ github.event.release.target_commitish }}
```

Comment:
`${{ github.event.release.target_commitish }}` is `main`?

Reply:
Yeah, and the tag is for npm https://github.com/microsoft/ts-gyb/runs/3469815943?check_suite_focus=true